src/windows.rs

```diff
@@ -124,6 +127,43 @@ pub fn list() -> Result<Vec<TrashItem>, Error> {
     }
 }
 
+pub fn metadata(item: &TrashItem) -> Result<TrashItemMetadata, Error> {
+    ensure_com_initialized();
+    unsafe {
```

Comment:
Could each unsafe function call get its own unsafe block? I understand that this is how most of the functions are done, but I don't want this practice to proliferate.
It's just a small step towards better `unsafe`, as I still wouldn't be able to reason about safety here 😅 (*nor do I have the expectation that this needs to happen here*).

For context, I [recently have been shown how to deal with `unsafe`](https://github.com/Byron/gitoxide/pull/1237), and I think it affected me 😁.

Reply:
I will address this in the next commit